core/client/sentry.js

```diff
@@ -0,0 +1,24 @@
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+
+export default function initSentryClient() {
+  const dsn = Meteor.settings.public.sentrySettings.dsn || null;
```

Comment:
We need to handle configs that do not define `sentrySettings` or we are going to break most lemverse instances :)